packages/Icon/README.md

```diff
@@ -0,0 +1,31 @@
+# Icon
+
+## Description
+
+`<Icon>` component is a graphic to represent something else
+
+## Installation
+
+```
+yarn add @paprika/icon
+```
+
+or with npm:
+
+```
+npm install @paprika/icon
+```
+
+## Usage
+
+If you need a custom icon for paprika, you can use the `@paprika/icon`
+
+```jsx
+import AddIcon from "@paprika/icon";
+
+<AddIcon />;
+```
+
+### Note
+
+If you are a consumer who needs a custom icon for your project, you have to use [wasabicons](https://design.wegalvanize.com/icons)
```

Comment:
can we make emphasis that this is only for galvanize users since wasabicons are only available internally.